pyworkflow/gui/project/project.py

```diff
@@ -146,23 +144,26 @@ def _onClosing(self):
             if not self.project.openedAsReadOnly():
                 self.saveSettings()
         except Exception as ex:
-            print("%s %s" %(Message.NO_SAVE_SETTINGS, str(ex)) )
+            print("%s %s" % (Message.NO_SAVE_SETTINGS, str(ex)))
         ProjectBaseWindow._onClosing(self)
      
     def loadProject(self):
-        self.project = pw.project.Project(self.projPath)
-        self.project.load()
+        proj = pw.project.Project(pw.Config.getDomain(), self.projPath)
```

Comment:
How is this releated to settings. I mean project is returned instead of set to self.project (as before) but settings is  not touched.

Do you see any issue with this?

Reply:
self.project is assigned in the __init__ with the return value from this function. This was a small refactoring...just PyCharm complaining that self.project was not initialized in the __init__. But everything should work as before.

Reply:
hehehe, if we are going to pay attention to those pycharm complaints ....we are doomed!! ;-)